packages/cli/lib/lib/babel-config.js

```diff
@@ -36,7 +36,7 @@ module.exports = function (env, options = {}) {
 		overrides: [
 			// Transforms to apply only to first-party code:
 			{
-				exclude: /(^|\/|\\)node_modules(\/|\/)/,
+				exclude: /node_modules/,
```

Comment:
Windows compatible exclude

Reply:
I think the second `\/` at the end of the old regex was maybe supposed to be a `\\`, but this should be fine as well

Reply:
lol yeah I thought that was odd. This should work:

```suggestion
				exclude: /(^|\/|\\)node_modules(\/|\\)/,
```

Reply:
Another option would be to use the match pattern:

```suggestion
				exclude: '**/node_modules/**',
```